templates/all_items.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Current To Do Items from Trello:</h2>
+           {% for cards_from_todo in cards_from_todo %}
+           <form action="{{url_for('complete_item')}}" method="POST">
+            <input type ='text' name='card_name' value="{{cards_from_todo['name']}}" readonly="readonly">
+            <input type = 'text' name="list_id" value="{{cards_from_todo['idList']}}" readonly="readonly">
```

Comment:
These 2 input fields do not need to be displayed to the end user.

Reply:
Will remove once I fix the add card feature back. Was using this to see that I was pulling the correct card & list details.